docs/examples/minimal_transfer_script.rst

```diff
@@ -51,3 +51,222 @@ For simplicity, the script will prompt for login on each use.
     task_doc = transfer_client.submit_transfer(task_data)
     task_id = task_doc["task_id"]
     print(f"submitted transfer, task_id={task_id}")
+
+
+Minimal File Transfer Script Handling ConsentRequired
+-----------------------------------------------------
+
+The above example works with certain endpoint types, but will fail if either
+the source or destination endpoint requires a ``data_access`` scope. This
+requirement will cause the Transfer submission to fail with a
+``ConsentRequired`` error.
+
+The example below catches the ``ConsentRequired`` error and retries the
+submission after a second login.
+
+This kind of "reactive" handling of ``ConsentRequired`` is the simplest
+strategy to design and implement.
+
+We'll also enhance the example to take endpoint IDs from the command line.
+
+.. code-block:: python
```

Comment:
For a runnable script like this, it might make sense to push this into an actual script, particularly for the purposes of running the code to test its functionality, and running it through the full suite of pre-commit hooks.

```rst
..  literalinclude:: consent_required_reactive.py
    :language: python
```

Reply:
I approved the PR, but still submit this feedback for your consideration. 😀

Reply:
I've been meaning to try to read up on "what was the syntax for including scripts again...?"

If I can change it without breaking the doc build in < 10 minutes, I'll make the change here and include it in the PR.

Reply:
I just got back to this and pushed a small change to remake these into a set of downloadable literalinclude targets.
I'd appreciate it if you could take a gander at the result and either approve again or merge (either is fine, IMO) if it looks good!